packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-testing.md

```diff
@@ -2,104 +2,110 @@
 
-<label><input type="checkbox" /> Is it understandable and usable? </label><br/>
-  - <label><input type="checkbox" /> When you navigate to an item by keyboard or by screen reader, you can easily understand and use the item. </label><br/>
-  - <label><input type="checkbox" /> The flow of information makes sense when navigating. Screen readers navigate the page in DOM order. If you used CSS to visually reposition elements, they might be announced in a nonsensical sequence. Instead, if you need something to appear earlier in the page, try to physically move it earlier in the DOM.</label>
+    <Checkbox label={<span> <b>Landmark</b> regions should have labels when there is more than one type of landmark, and they are not the same (such as navigation, main, form, etc.). Section element shouldn’t be used unless it has a label.</span>} id="general-criteria-8f" />
 
-**Accessibility points** <br />
-<label><input type="checkbox" /> **Structure:** The visual information architecture should map to the various rotor menus that exist by default. </label><br />
-- <label><input type="checkbox" /> **Check rotor:** Are there good headings, landmarks, links, form controls, and other elements? Do they make sense, and are they descriptive without overloading information? </label><br />
-  - <label><input type="checkbox" /> Headings: Check that heading levels convey structure/content and do not skip. A common practice is to use a single h1 for the primary headline or logo on a page, h2s for designating major sections, and h3s for supporting sections. </label><br />
-  - <label><input type="checkbox" /> Links </label><br />
-  - <label><input type="checkbox" /> Form controls </label><br />
-  - <label><input type="checkbox" /> Landmarks </label><br />
-  - <label><input type="checkbox" /> Tables </label><br />
+    <Checkbox label={<span><b>Tables</b> and table content is clearly described. <a href="https://webaim.org/techniques/tables/">WebAIM has additional guidance on this.</a></span>} id="general-criteria-8g" />
 
+    <Checkbox label={<span><b>Aria</b> provides descriptive details to screen reader users, without reiterating or overriding the text that is already there. If there is visible text then there doesn’t need to be an <span className="ws-code">aria-label</span>. </span>} description={<span>It’s important to understand that ARIA can only affect the semantics of an element; not the behavior. For example, while you can hide an element from screen readers with <span> className="ws-code">aria-hidden=”true”</span>, that does not change the focus behavior for that element.</span>} id="general-criteria-8h" />
```

Comment:
@jessiehuff wdyt about tweaking this verbiage? Feel like we could name `aria-label` as a way to add a non-visual label to certain elements, and `aria-labelledby` to use an existing text or aria-label to label another element. 

Though if this is meant to be more of a general point about aria rather than strictly labeling, maybe the verbiage about when you don't need an aria-label could be its own bullet?

Additionally, looks like the description has a typo in the description. `<span> className="ws-code">aria-hidden=”true”</span>` should be `<span className="ws-code">aria-hidden=”true”</span>`

Reply:
Yeah, it was originally intended as more of a general statement of when something needs to be labeled vs. not, but admittedly I probably shouldn't have just singled out `aria-label` in that circumstance. (I started calling this out because I noticed that once people started trying to improve their a11y, they could at times just go crazy adding `aria-label`s even when there was no need for one and adding one might even make the experience worse.) Maybe we could change the `aria-label` in this context to a label in general and then call out your point in a bullet.